pom.xml

```diff
@@ -25,6 +25,7 @@
 	</properties>
 	<modules>
 		<module>spring-cloud-stream</module>
+		<module>spring-cloud-stream-rxjava</module>
```

Comment:
It would be nice to find a separate repo for these kind of projects. May be `spring-cloud-stream-extensions` ?